app/scripts/controllers/user-storage/encryption.ts

```diff
@@ -0,0 +1,117 @@
+import sjcl from 'sjcl';
```

Comment:
This library seems to be unmantained, was it chosen for any particular reason?

Reply:
No preference on the library - was chosen from using this snippet:
https://github.com/luke-park/SecureCompatibleEncryptionExamples/blob/master/TypeScript/SCEE-sjcl.ts

We are not tied to it so can switch it out 😄

Reply:
Great find, and this has been switched out!